org.civicrm.reqangular/js/test/models/contact.spec.js

```diff
@@ -174,6 +179,45 @@ define([
           });
         });
 
+        describe('when filter includes a period for contracts', function () {
+          var result, sampleContact;
+          var sampleDates = {
+            from: '2018-01-01 00:00:00',
+            to: '2018-12-31 23:59:59'
+          };
+
+          beforeEach(function (done) {
+            sampleContact = _.find(contactData.all.values, {
+              id: _.first(jobContractData.contactsWithContractsInPeriod.values).id });
+
+            spyOn(jobContractAPI, 'getContactsWithContractsInPeriod').and.callThrough();
+            Contact.all({
+              with_contract_in_period: [
+                sampleDates.from,
+                sampleDates.to
+              ]
+            })
+              .then(function (_result_) {
+                result = _result_;
+
+                done();
+              });
+            $rootScope.$digest();
+          });
+
+          it('calls the job contract API', function () {
+            expect(jobContractAPI.getContactsWithContractsInPeriod)
+              .toHaveBeenCalledWith(
+                sampleDates.from,
```

Comment:
what's the reason for this formatting? This
```js
expect(jobContractAPI.getContactsWithContractsInPeriod)
  .toHaveBeenCalledWith(sampleDates.from, sampleDates.to)
```
is still readable and takes less room

Reply:
I think I had some longer code and then I just auto-replaced it. Done.